src/OrchardCore.Modules/OrchardCore.Lists/Controllers/OrderController.cs

```diff
@@ -53,7 +53,12 @@ public async Task<IActionResult> UpdateContentItemOrders(string containerId, int
             }
 
             // Include draft items.
-            var pageOfContentItems = (await _containerService.QueryContainedItemsAsync(containerId, true, pager, false)).ToList();
```

Comment:
See how the comment on this code says include draft items (line 55).

here it isn't, it's including the `None` status, as that is the default.

Another reason to remove the `None` status, it would result in strange results.

Reply:
Hmm, now that there is no `None`, some kind of filter will always be applied and none of the existing ones (`Published`, `Draft`, `Owner`) will return both published and draft items. I'll add a `Latest` then.